data/remnant missions.txt

```diff
@@ -1252,3 +1252,257 @@ mission "Remnant: Expanded Horizons Quarg 3"
+	minor
+	landing
+	repeat
+	to offer
+		has "remnant blood test pure"
+		not "Remnant: Broken Jump Drive 1: active"
+		not "Remnant: Broken Jump Drive 1: done"
+	source
+		government "Remnant"
+		attributes outfitter
+	destination
+		government "Remnant"
+		attributes outfitter
+		not distance 0
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			branch repeated
+				has "Remnant: Broken Jump Drive 1: offered"
+			`As the <ship> settles onto its designated landing pad, you see someone emerge from the nearby outfitter and quickly make their way to you. The man waits a safe distance before approaching once you cycle the hatch and disembark. "Greetings, Captain <last>," trills the Remnant, "we saw the news that the newcomer among us was tracking down an erratic Korath ship. When you came in to land, we saw on the scans that you salvaged its malfunctioning jump drive. Could we take it to study?"`
+			choice
+				`	"Maybe later."`
+					goto refuse
+				`	"That would be a good use for it."`
+					goto accept
+			
+			label repeated
+			`You have a broken jump drive that the Remnant might be interested in. Do you want to seek out someone to give it to?`
+			choice
+				`	(Maybe later.)`
+					defer
+				`	(Yes.)`
+			`	You find a Remnant director and inform them about the broken jump drive that you have.`
+			
+			label accept
+			`	"Excellent!" he glances at his datapad. "There are several teams working on aspects of the jump drive, and the team with the highest priority for this one is on <planet>. I will notify them to expect you."`
+				accept
+			label refuse
+			`	He considers this, and sings, "If you wish. Our offer stands, whenever you choose to return."`
+				decline
+	on complete
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 300000
+		"reputation: Remnant" ++
+		conversation
+			`When you arrive on <planet> there is a team of engineers and mechanics waiting for you, along with a flatbed freight truck. Once you open the cargo doors they quickly free the broken jump drive from its moorings and load it delicately onto their truck. The lead engineer comes over to hand you <payment>. "Thank you for salvaging this. I suspect my compatriot who spoke to you never mentioned this, but we have a fund set aside for rewarding captains who bring in rare pieces of alien technology. If you find any more of these drives, there is sure to be a research team somewhere that is willing to reward you for it."`
+			choice
+				`	"I will keep a lookout for more."`
+					goto end
+				`	"Aren't you going to replace it with a hyperdrive?"`
+			`	The engineer looks at you incredulously. "You mean you were actually trying to use this drive?" He shakes his head in disbelief. "You are lucky you managed to get this far, then. It is surprising that they exist at all, so it is too much to expect them to work too. If you need a replacement drive, there are plenty available in the outfitter.`
```

Comment:
Is this really a valid question anymore if the broken JD can't be used?

Reply:
I left it in on the grounds that I'm sure someone, somewhere is going to want the option to ask them to replace it with a hyperdrive. Or just want to see the reaction of the engineer when they imply they were flying with it installed.

Reply:
Yeah, the reaction is good. It just seems like an odd question now.

Reply:
Most players will probably skip it, but for those who are greedy and those who have done the Unfettered missions, it could be a valid question.

Regarding his response `It is surprising that they exist at all, so it is too much to expect them to work too.` would it be better if I took out the "so" and added a semi-colon?

`It is surprising that they exist at all; it is too much to expect them to work too.`